### PR TITLE
Adding chia peer command

### LINKED ISSUE #12298: CLI naming
### What happened?

I propose renaming `chia show` to `chia node` since half the commands are not about showing information but about interacting with the full node.
Or only the commands related to doing actions on the full node could be moved to `chia node`, while other commands that are about showing information like `chia netspace` could be moved to a `chia show` subcommand.

Second thing, the 2 letter short options are a footgun for people used to CLI tools. The UNIX convention is for short options to be only 1 letter long so that they can be combined in one argument, but for example `chia show` has 3 different options `-bh`, `-b` and `-h`.

This might be low priority but the CLI is often the first point of contact for beginners, so reducing confusion is a good thing. I was personally confused by it. 

### Version

1.4.0

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by wallentx**
Addition of chia peer command for (wallet, farmer, full node and harvester) & slight refactor of chia show.

### CODE REVIEW

File: chia/cmds/peer.py
Comment (altendky) [inline chia/cmds/peer.py:8]: If you need the `time` module, go ahead and import it.

```suggestion
async def print_connections(client, NodeType, trusted_peers: Dict) -> None:
```

File: chia/cmds/peer.py
Comment (altendky) [inline chia/cmds/peer.py:78]: I think this is what comes out of the config?

```suggestion
            trusted_peers: Dict[str, str] = config["full_node"].get("trusted_peers", {})
```

**Comment by jack60612** [comment]
Testing locally now

**Comment by jack60612** [comment]
Local CLI tests successful!!

**Comment by jack60612** [comment]
> This pull request **introduces 8 alerts** when merging [d398434](https://github.com/Chia-Network/chia-blockchain/commit/d3984347973b3989a1827e735bbe59b189bdd7d3) into [6923ad8](https://github.com/Chia-Network/chia-blockchain/commit/6923ad8e12a376e7f5fa428a0cea0de70ce53e7f) - [view on LGTM.com](https://lgtm.com/projects/g/Chia-Network/chia-blockchain/rev/pr-d254e07bc975b818c0b621089bdf3fb6b4bf2367)
> 
> **new alerts:**
> 
> * 8 for Module-level cyclic import

False alarm

**Comment by emlowe** [comment]
This break backwards compatibility right - `chia show -c` doesn't work anymore?  Should we maintain the `chia show` options? At the very least I think `chia show -c` should print that it is deprecated and use `chia peer -c`?

**Comment by jack60612** [comment]
> This break backwards compatibility right - `chia show -c` doesn't work anymore? Should we maintain the `chia show` options? At the very least I think `chia show -c` should print that it is deprecated and use `chia peer -c`?

i can add that. It does break backwards compatibility

File: chia/cmds/show.py
Comment (emlowe) [inline chia/cmds/show.py:232]: breaking this backwards compat should be avoided (I understand the multi-letter options are a problem, but we shouldn't just break things)

**Comment by emlowe** [comment]
You also need to fix `chia show -a` and `chia show -r` and anythign else removed from `chia show`

For example, I know the `--wallet-rpc-port` is unused, but by removing it you break scripts that were using it accidentally - it would be better to leave it in and make the help say it's unused and deprecated. Oh, click has a deprecated flag https://click.palletsprojects.com/en/8.1.x/api/?highlight=deprecated#click.Command

**Comment by jack60612** [comment]
> You also need to fix `chia show -a` and `chia show -r` and anythign else removed from `chia show`
> 
> For example, I know the `--wallet-rpc-port` is unused, but by removing it you break scripts that were using it accidentally - it would be better to leave it in and make the help say it's unused and deprecated. Oh, click has a deprecated flag https://click.palletsprojects.com/en/8.1.x/api/?highlight=deprecated#click.Command

ok that makes sense.

**Comment by jack60612** [comment]
@emlowe is it good now?

**Comment by jack60612** [comment]
all fixed

File: chia/cmds/show.py
Comment (emlowe) [inline chia/cmds/show.py:338]: This needs to be left in - it's at least used later on.

Comment (jack60612) [reply chia/cmds/show.py:338]: This was accidently removed, i'm going to re review this whole pr before making any more changes

Comment (jack60612) [reply chia/cmds/show.py]: Sorry about the review spam, should be good now

**Review by mariano54 — CHANGES_REQUESTED** [review]
I think the `chia peer` command should support any service (farmer, harvester, wallet etc), since they all support the same RPC calls.

This would make it more generic and useful for all types of situations. `chia show` was meant to be just for the node, but if we are refactoring this RPC, I think we should have a higher standard and make it generalizeable.

**Comment by jack60612** [comment]
both the chia peer and the chia show command were successfully tested locally.

File: chia/cmds/cmds_util.py
Comment (mariano54) [inline chia/cmds/cmds_util.py:43]: I would rename to `execute_with_any_service`. Node means fulls node in chia vocabulary.

**Review by mariano54 — DISMISSED** [review]
LGTM apart from small nit for naming

File: chia/cmds/cmds_util.py
Comment (jack60612) [reply chia/cmds/cmds_util.py]: Renamed!

**Review by mariano54 — DISMISSED** [review]

**Review by mariano54 — CHANGES_REQUESTED** [review]
Need to fix the LGTM issues (cyclic import)

**Comment by jack60612** [comment]
> Need to fix the LGTM issues (cyclic import)

they are wrong, i dont know what they are yapping about.

**Comment by emlowe** [comment]
Yea, LGTM has some issues - Kyle would actually like us to stop using it as it's not adding much value

**Comment by mariano54** [comment]
We can't merge this PR if it adds 40 LGTM issues. 
Or should we just start ignoring it completely?

**Comment by jack60612** [comment]
closing this pr in favor of #12532

**Pull request closed without merging**
